docs/cli/graphql-transformer/directives.md

```diff
@@ -476,70 +478,126 @@ type Post
+You can use the `operations` argument to specify which operations are enabled as follows:
 
 - **read**: If the record's owner is not the same as the logged in user (via `$ctx.identity.username`), throw `$util.unauthorized()` in any resolver that returns an object of this type.
-- **create**: Inject the logged in user's `$ctx.identity.username` as the *ownerField* automatically.
-- **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
-- **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
+- **create**: Inject the logged in user's `$ctx.identity.username` as the `ownerField` automatically.
+- **update**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
+- **delete**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
 
-**Note**: When specifying operations as a part of the @auth rule, the operations not included in the list are not protected by default. For example, let's say you have the following schema:
+You must ensure that the `create` operations rule is either specified directly or indirectly to ensure that the owner's identity is stored with the record so it can be verified on subsequent requests.
+
+```graphql
+# owner identity automatically stored on every object
+type Post @model @auth(rules: [{ allow: owner }]) {
+  id: ID!
+  title: String!
+}
+
+# owner identity automatically stored on every object
+type Post @model @auth(rules: [{ allow: owner, operations: [create] }]) {
+  id: ID!
+  title: String!
+}
+
+# owner identity not stored on objects
+type Post @model @auth(rules: [{ allow: owner, operations: [read] }]) {
+  id: ID!
+  title: String!
+}
+```
+
+<amplify-callout>
+
+When specifying operations as a part of the `@auth` rule, the operations not included in the list are not protected by default.
+
+</amplify-callout>
+
+
+Let's take a look at a few examples:
 
 ```graphql
 type Todo @model
-  @auth(rules: [{ allow: owner, operations: [read] }]) {
+  @auth(rules: [{ allow: owner }]) {
   id: ID!
   updatedAt: AWSDateTime!
   content: String!
 }
 ```
 
-In this schema, only the owner of the object has the authorization to perform read (getTodo and listTodos) operations on the owner created object. But this does not prevent any other owner (any user other than the creator or owner of the object) to update/delete some other owner's object.
-Here's a truth table for the above-mentioned schema. In the table below `other` refers to any user other than the creator or owner of the object.
+In this schema, only the owner of the object has the authorization to perform read (`getTodo` and `listTodos`), update (`updateTodo`), and delete (`deleteTodo`) operations on the owner created object. This prevents the object from being updated or deleted by users other than the creator of the object.
```

Comment:
I wonder if the spirit of this example is lost. Sounds like the original intention was to show an example where users could create their own objects and other users could just do get/list against it.

Reply:
Ok reading further, I think you address this below.